### ISSUE TITLE
Change to underlying file from a symbolic link doesn't update CSV display

### ISSUE THREAD

**Issue by rickhg12hs** [jupyterlab/jupyterlab#4944]
Displaying a CSV file from a symbolic link doesn't update the display when the underlying file changes.

For example,
```bash
$ ln -s some_real_CSV_file my_local_link.csv
```
Display `my_local_link.csv` in jupyterlab (looks fine initially) ... followed by modifying `some_real_CSV_file`,
```bash
$ some_program > some_real_CSV_file
```
The CSV display in jupyterlab doesn't change (even when `rm some_real_CSV_file`)

**Comment by blink1073** [comment]
Hi @rickhg12hs, I suspect this is a stale-ness issue.  We don't get notifications in the server when something has changed.  If you close the file in jlab and re-open it, do you see the modifications?

**Comment by rickhg12hs** [comment]
> If you close the file in jlab and re-open it, do you see the modifications?

Yes.

In addition, if the underlying file is deleted, within 5 seconds the file listing in the "Left Area" updates shows this.  Perhaps a pop-up, or something, to let the user know that the CSV file is gone/changed/stale/? would be useful.

**Comment by blink1073** [comment]
Yes, the file listing is polling the server every 10 seconds for changes.  I agree with the need for a pop up when we see that an open file is gone.
